pronto_quadruped_ros/include/pronto_quadruped_ros/legodo_handler_ros.hpp

```diff
@@ -45,22 +45,24 @@ class LegodoHandlerBase {
 public:
     EIGEN_MAKE_ALIGNED_OPERATOR_NEW
 public:
-    using  LegScalarMap = LegDataMap<double>;
+    using LegScalarMap = LegDataMap<double>;
     using Update = RBISUpdateInterface;
 
 public:
     LegodoHandlerBase(ros::NodeHandle& nh,
                       StanceEstimatorBase& fcf,
                       LegOdometerBase& fj);
+    virtual ~LegodoHandlerBase() = default;
 
 protected:
     StanceEstimatorBase& stance_estimator_;
     LegOdometerBase& leg_odometer_;
 
-    Eigen::Vector3d r_legodo;
-    Eigen::Vector3d R_legodo_init;
-    Eigen::Vector3d r_legodo_init;
-    Eigen::Matrix3d cov_legodo;
+    std::string base_link_name_;           ///< Name of the base_link
+    std::vector<std::string> foot_names_;  ///< Name of the feet frames (in LF, RF, LH, RH order)
+
```

Comment:
where did the `_init` members go?

Reply:
They were unused in the code themselves... one is now local to the constructor, the other was unused.